RELEASES.md

```diff
@@ -0,0 +1,131 @@
+or comment to request its inclusion.
+
+## 🎯 Major and minor releases
+
+Major and minor releases of moby will be made from main. Releases will be marked
+with GPG signed tags and posted at https://github.com/moby/moby/releases. The
+tag will be of the format `v<major>.<minor>.<patch>-[suffix[N]]`.
+
+After a minor release of the main module, a branch will be created, with the format
+`release/<major>.<minor>` from the minor tag. All further patch releases will be
+done from that branch. For example, after the release of `v2.1.0`, a branch
+`release/2.1` would be created from that tag. All future patch releases will be
+done against that branch.
+
+## 🛠️ Patch releases
+
+Patch releases are made directly from release branches and will be done as needed
+by the release branch owners.
+
+> [!IMPORTANT] 
+> Security releases are also "patch releases", but follow a
+> different procedure. Security releases may be developed in a private repository,
+> released and tested under embargo before they become publicly available.
+
+## 🎬 Pre-releases
+
+Pre-releases, such as alphas, betas and release candidates will be conducted
+from their source branch. For major and minor releases, these releases will be
+done from `master`. For patch releases, it is uncommon to have pre-releases but
+they may have an `rc` based on the discretion of the release branch owners.
+
+## 🏗️ Modules
+
+The Moby project is a collection of modules, each with its own versioning but with
+the same minor release cadence.
+
+- `github.com/moby/moby/v2`: The main module, which contains the core components of Moby.
+  This module is used to build the main daemon is not intended for direct use by clients.
+- `github.com/moby/moby/api`: The API module, which contains the API types and documentation.
+- `github.com/moby/moby/client`: The client module, which contains the client library for
+  interacting with the Moby API.
+
+## Support Horizon
+
+Support horizons will be defined corresponding to a release branch, identified
+by `<major>.<minor>`. Release branches will be in one of several states:
+
+- __*Future*__: An upcoming scheduled release.
+- __*Alpha*__: The next scheduled release on the main branch under active development.
+- __*Beta*__: The next scheduled release on the main branch under testing. Begins 2-4 weeks before a final release.
+- __*RC*__: The next scheduled release on the main branch under final testing and stabilization. Begins 1 week before a final release. For new releases where the source branch is main, the main branch will be in a feature freeze during this phase.
+- __*Active*__: The release is a stable branch which is currently supported and accepting patches.
+- __*Extended*__: The release branch is only accepting security patches.
+- __*End of Life*__: The release branch is no longer supported and no new patches will be accepted.
+
+Releases will be supported until at least the next _minor_ release. Any minor release
```

Comment:
I can do some follow ups, but try to avoid future tense in docs. While this is for sure "for future events", I think most parts still work well (and more to the point) in present tense.